CHANGELOG.md

```diff
@@ -15,6 +15,22 @@ x.x.x Release notes (yyyy-MM-dd)
  ### Internal
 * None.
 
+2.19.1 Release notes (yyyy-MM-dd)
```

Comment:
Please don't specify version number. `npm set-version` will do it during a release.